prism/src/main/java/org/apache/falcon/resource/proxy/ExtensionManagerProxy.java

```diff
@@ -338,15 +337,15 @@ public APIResult submit(
         return new APIResult(APIResult.Status.SUCCEEDED, "Extension job submitted successfully:" + jobName);
     }
 
-    private Map<EntityType, List<Entity>> getEntityList(String extensionName, String jobName,
+    private TreeMap<EntityType, List<Entity>> getEntityList(String extensionName, String jobName,
```

Comment:
Fair enough, as long as it is an Interface as you have used (SortedMap) and not the impl., it is fine.